app/Http/Controllers/Webapp/PostController.php

```diff
@@ -0,0 +1,1890 @@
+                , 'danger');
+            $recoverChecksum = $clientChecksum;
+        }
+
+        $site = Site::find($post->site_id);
+        $additionCss = (!empty($site)) ? $site->getAdditionCss() :null;
+        $draftUnsaved = PostHistory::getFirstFrom($post, PostHistory::STATUS_DRAFT, $post->updated_at->toDateTimeString());
+
+        return view('webapp/posts/edit'.(($readOnly) ? '_readonly' : ''), [
+            'post' => $post,
+            'lockStatus' => $lockStatus,
+            'sites' => Site::where('status', 1)->get(),
+            'categoriesSet' => $categoriesSet,
+            'subkws' => $subkws,
+            'mainkw' => $mainkw,
+            'action_delete' => $action_delete,
+            'additionCss' => $additionCss,
+            'postChecksum' => $clientChecksum,
+            'recoverChecksum' => $recoverChecksum,
+            'draftUnsaved' => $draftUnsaved,
+            'readOnly' => $readOnly
+        ]);
+    }
+
+    public function postEdit(Request $request, $postId = 0) {
+        if(empty($post = Post::find($postId)) || !$post->canEditBy($this->currentUser) ) {
+            return '404 Not Found';
+        }
+
+
+        if($post->isLock($this->currentSession)) {
+            $this->_flashMessage($request, 'このコンテンツは現在ロックされています。編集できません。', 'danger');
+            return redirect()->to(action('Webapp\\PostController@getEdit', $postId))->send();
+        }
+        if($request->get('site_id') == ''){
+            $this->_flashMessage($request, 'ターゲットサイトを選んでください', 'danger');
+            return redirect()->to(action('Webapp\\PostController@getEdit', $postId))->send();
+        }
+
+        $postTitle = $request->get('post_title');
+        $postContent = $request->get('post_content');
+        $postDescription = trim($request->get('post_description'));
+
+        $previousContent = $post->content;
+        $post->title = !empty($postTitle) ? $postTitle : 'タイトルなし';
+        $post->content = $postContent;
+        $post->category = $request->get('category');
+        $post->description = $postDescription;
+        $post->feature_img = !empty($request->get('new_feature_img_url')) ? $request->get('new_feature_img_url') : $request->get('feature_img_url');
+
+        $post->priority = (int)$request->get('post_priority', 0);
+        $post->word_count = (int)$request->get('word_count', 0);
+
+        $submitType = strpos($request->get('submit_type'),'下書き') !== false ? 'draft' : 'publish';
+
+        if($submitType == 'publish' && $this->currentUser->is(User::CONTRIBUTOR)){
+            return '404 Not Found';
+        }
+
+        $oldSiteId = $post->site_id;
+        $oldTargetPostId = $post->target_post_id;
+        $oldPostId = 0;
+        if($post->site_id == 0 || $oldSiteId != $request->get('site_id')) {
+            $post->site_id = $request->get('site_id');
+        }
+
+        try {
+//            $action = ($submitType == 'publish' && $post->published_status == 0 && $post->target_post_id == 0) ? 'publish' : 'edit';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.relipa.blog/coding_rules#rule_key=php%3AS125)